classes/class-student-blog-handler.php

```diff
@@ -156,7 +140,9 @@ protected function get_blog_id_for_path() {
 
 		$path = '/' . $path . '/';
 
-		return get_blog_id_from_url( $this->domain, $path );
+		$blog_id = get_blog_id_from_url( $this->domain, $path );
```

Comment:
There was a bit of general refactoring around the area I was working on, as the student blog handler was repeating a function in the base class, but with one difference. One part of that was consistency when no blog id was returned from a function, it was null some places, and int elsewhere.